src/components/KModal/KModal.vue

```diff
@@ -7,12 +7,14 @@
     role="dialog"
   >
     <div
+      ref="modalOuter"
       class="k-modal-backdrop modal-backdrop"
       @click="(evt: any) => close(false, evt)"
     >
       <FocusTrap
         ref="focusTrap"
         :active="false"
+        :fallback-focus="modalOuter?.$el"
```

Comment:
if this can be undefined, how does using this element as a fallback resolve the issue?

Reply:
it's null until component is mounted

Reply:
so then why is the optional chaining needed if it's there on mount?